index.bs

```diff
@@ -262,10 +252,10 @@ When asked to <dfn abstract-op>remove client hints from redirect if needed</dfn>
   <var>hintName</var> and if the result of running <a
   href="https://w3c.github.io/webappsec-permissions-policy/#algo-should-request-be-allowed-to-use-feature">Should
   request be allowed to use feature?</a>, given <var>request</var> and <var>hintName</var>’s
-  <a href="http://httpwg.org/http-extensions/client-hints.html#opt-in-via-feature-policy">associated
+  <a href="#policy-controlled-features">associated
```

Comment:
This change in particular (as well as the similar change just above) could use some extra eyeballs -- it seemed like a mistake, but I could be wrong!

Reply:
I'm failing to find what this was pointing to... Changing the link as you did makes sense.